lib/types/terminus.js

```diff
@@ -1,17 +1,40 @@
 'use strict';
 var types = require('../types'),
+    errors = require('../errors'),
     Terminus = module.exports = {};
 
+function terminusDurability(value) {
```

Comment:
If you want to be consistent, we should only accept ['none', 'configuration', 'unsettled-state'] and default to 'none' below.